apps/docs/content/components/PricingOptions.mdx

```diff
@@ -0,0 +1,747 @@
+      Code completions, Chat, and more for indie developers and freelancers.
+    </PricingOptions.Description>
+    <PricingOptions.Price
+      currencySymbol="$"
+      trailingText="per month / $100 per year"
+    >
+      10
+    </PricingOptions.Price>
+    <PricingOptions.PrimaryAction href="/buy">
+      Buy now
+    </PricingOptions.PrimaryAction>
+    <PricingOptions.SecondaryAction href="/contact">
+      Contact sales
+    </PricingOptions.SecondaryAction>
+  </PricingOptions.Item>
+</PricingOptions>
+```
+
+### Two items
+
+This is the two-item layout for the pricing options component.
+
+```jsx live
+<PricingOptions>
+  <PricingOptions.Item>
+    <PricingOptions.Label>Recommended</PricingOptions.Label>
+    <PricingOptions.Heading>Copilot</PricingOptions.Heading>
+    <PricingOptions.Description>
+      Copilot in the coding environment.
+    </PricingOptions.Description>
+    <PricingOptions.Price
+      currencySymbol="$"
+      trailingText="per month / $100 per year"
+    >
+      10
+    </PricingOptions.Price>
+    <PricingOptions.FeatureList>
+      <PricingOptions.FeatureListItem>
+        Everything in Copilot Business plus:
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Chat in IDE and Mobile
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        CLI assistance
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Code completions
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+    <PricingOptions.PrimaryAction href="/buy">
+      Buy now
+    </PricingOptions.PrimaryAction>
+    <PricingOptions.SecondaryAction href="/contact">
+      Contact sales
+    </PricingOptions.SecondaryAction>
+  </PricingOptions.Item>
+
+  <PricingOptions.Item>
+    <PricingOptions.Label>Recommended</PricingOptions.Label>
+    <PricingOptions.Heading>Copilot Business</PricingOptions.Heading>
+    <PricingOptions.Description>
+      Copilot personalized to your organization throughout the software
+      development lifecycle. Requires GitHub Enterprise Cloud.
+    </PricingOptions.Description>
+    <PricingOptions.Price currencySymbol="$" trailingText="per user / month">
+      39
+    </PricingOptions.Price>
+    <PricingOptions.FeatureList>
+      <PricingOptions.FeatureListItem>
+        Everything in Copilot Business plus:
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Chat in IDE and Mobile
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        CLI assistance
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Code completions
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+    <PricingOptions.PrimaryAction href="/buy">
+      Join waitlist
+    </PricingOptions.PrimaryAction>
+  </PricingOptions.Item>
+</PricingOptions>
+```
+
+### Three items
+
```

Comment:
The three option example looks quite squished in docs, so can we put a note to refer to Storybook for this example instead, similar to how we do it in AnchorNav docs?
<img width="950" alt="Screenshot 2024-03-20 at 11 43 43" src="https://github.com/primer/brand/assets/13340707/6a277522-33c0-452a-860c-7a86de648489">

<img width="779" alt="Screenshot 2024-03-20 at 11 43 13" src="https://github.com/primer/brand/assets/13340707/b03b5001-9bfd-460b-8e5f-0d2b73a4a1f2">